src/components/Todo.tsx

```diff
@@ -0,0 +1,66 @@
+import { Container } from '@material-ui/core';
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  Link,
+} from '@material-ui/core';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import React, { FC } from 'react';
+
+type TodoProps = {
+  id: number;
+  title: string;
+  description: string;
+  deadline: Date;
+  categories: {
+    id: number;
+    category: string;
+  }[];
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      width: '100%',
+    },
+    heading: {
+      fontSize: theme.typography.pxToRem(15),
+      fontWeight: theme.typography.fontWeightRegular,
+    },
+  })
+);
+
+export const Todo: FC<TodoProps> = (props: TodoProps) => {
+  const classes = useStyles();
+
+  console.log(props.categories);
+  return (
+    <Accordion>
+      <AccordionSummary aria-controls="panel1a-content" id="panel1a-header">
+        <Typography className={classes.heading}>{props.title}</Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Typography>{props.description}</Typography>
+      </AccordionDetails>
+      <AccordionSummary>
+        <Typography>Deadline : {props.deadline.toString()}</Typography>
+      </AccordionSummary>
+      <AccordionDetails>
+        <Typography>Categories: </Typography>
+        {props.categories.map((data, index) => {
+          return (
+            <Container key={index}>
+              <Typography>
+                <Link href={'/categories/' + data.id.toString()}>
```

Comment:
基本的に template literal 使ってます！
ref: https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html#inference-with-template-literals
```suggestion
                <Link href={`/categories/${data.id.toString()}`}>
```

Reply:
あぁはい、ありがとうございます。template literalを使いたい時「'」より「`」を使うことが忘れました🙇‍♂️　